crates/bevy_render/src/render_phase/draw_state.rs

```diff
@@ -283,6 +283,166 @@ impl<'a> TrackedRenderPass<'a> {
             .draw_indexed_indirect(indirect_buffer, indirect_offset);
     }
 
+    /// Dispatches multiple draw calls from the active vertex buffer(s) based on the contents of the `indirect_buffer`.
+    /// `count` draw calls are issued.
+    ///
+    /// The active vertex buffers can be set with [`TrackedRenderPass::set_vertex_buffer`].
+    ///
+    /// `indirect_buffer` should contain `count` tightly packed elements of the following structure:
+    ///
+    /// ```rust
+    /// #[repr(C)]
+    /// struct DrawIndirect {
+    ///     vertex_count: u32, // The number of vertices to draw.
+    ///     instance_count: u32, // The number of instances to draw.
+    ///     first_vertex: u32, // The Index of the first vertex to draw.
+    ///     first_instance: u32, // The instance ID of the first instance to draw.
+    ///     // has to be 0, unless [`Features::INDIRECT_FIRST_INSTANCE`] is enabled.
+    /// }
+    /// ```
+    pub fn multi_draw_indirect(
+        &mut self,
+        indirect_buffer: &'a Buffer,
+        indirect_offset: u64,
+        count: u32,
+    ) {
+        trace!(
+            "multi draw indirect: {:?} {}, {}x",
+            indirect_buffer,
+            indirect_offset,
+            count
+        );
+        self.pass
+            .multi_draw_indirect(indirect_buffer, indirect_offset, count);
+    }
+
+    /// Disptaches multiple draw calls from the active vertex buffer(s) based on the contents of the `indirect_buffer`.
+    /// The count buffer is read to determine how many draws to issue.
+    ///
+    /// The indirect buffer must be long enough to account for `max_count` draws, however only `count` will
+    /// draws will be read. If `count` is greater than `max_count`, `max_count` will be used.
```

Comment:
What does `count` refer to? Draws are issued; "draws will be read" is inconsistent. I want to fold the second sentence into the first; maybe "`count` draws, capped by `max_count`, will be issued".